### ISSUE TITLE
Music affected by environment

### ISSUE THREAD

**Issue by matshou** [yooksi/pz-icdp#4]
The environment the player is located in affects the acoustics of CD player music played.

For example, when the playable cd's are played in the bathroom/toilet the music sound gets distorted by heavy reverb. 
The reverb preset can be seen in the LUA output log and is perhaps affected by the type or size of the room.

I don't know if it is possible to disable this effect for sound played by ICDP, but we should look into this.

**Comment by Nebula-ru** [comment]
Yes, I noticed this for a long time, but so far I have no idea where to fix it.
